scripts/documentation/open-graph/generate-images.ts

```diff
@@ -146,6 +159,19 @@ function createOpenGraphImage(
   });
 }
 
+function copyImage(
+  backgroundImagePath: string,
+  targetFolder: string,
+  filename: string
+) {
+  const splits = backgroundImagePath.split('.');
+  const extension = splits[splits.length - 1];
+  copyFileSync(
+    backgroundImagePath,
+    resolve(targetFolder + `/${filename}.${extension}`)
```

Comment:
this might lead to an issue on Windows machines I believe. I'd use `join`. Or 

```
resolve(targetFolder, `./${filename}.${extension}`)
```

might work too (note the `,` and the 2nd part needs to start with a relative path I believe if resolve is being used).